python/torch_mlir_e2e_test/linalg_on_tensors_backends/refbackend.py

```diff
@@ -166,6 +166,7 @@ def invoke(*args):
 LOWERING_PIPELINE = ",".join([
     # Bufferize.
     "builtin.func(scf-bufferize)",
+    "builtin.func(torch-mlir-tm-tensor-bufferize)",
```

Comment:
nit: technically "tm" in tm-tensor is already "torch-mlir", so it's a bit redundant to have torch-mlir prefix on the pass name.